blocks/columns/columns.css

```diff
@@ -23,6 +23,7 @@
   display: flex;
   align-items: flex-start;
   flex-direction: column;
+  gap: 1em;
```

Comment:
Any specific reason for using `em` ? Typically we use `rem`.

Reply:
no specific reason. I see in original site they have used 'em' for this gap not 'rem' so followed the same here